src/server/repository/assessment/assessment/updateAssessment.ts

```diff
@@ -0,0 +1,24 @@
+import { Objects } from '@utils/objects'
+
+import { Assessment } from '@meta/assessment'
+
+import { BaseProtocol, DB } from '@server/db'
+import { read } from '@server/repository/assessment/assessment/read'
+
+export const updateAssessment = async (
+  params: {
+    assessment: Assessment
+  },
+  client: BaseProtocol = DB
+): Promise<Assessment> => {
+  const { assessment } = params
+
+  const assessmentUpdated = await client.one<Assessment>(
+    `update assessment set props = $1::jsonb where id = $2 returning  *;`,
+    [JSON.stringify(assessment.props), assessment.id],
+    (assessment) => {
+      return { ...Objects.camelize(assessment), metaCache: assessment.metacache }
+    }
```

Comment:
I think this is typo, should it be `metaCache: assessment.meta_cache`?
Also, I would support updating `metaCache` like you did for props`, but @minotogna can verify if this is needed

Reply:
You are right. There is a typo, actually, it should be `assessment.metaCache`

Reply:
It should be the property of the database table